edge/pkg/metamanager/process.go

```diff
@@ -284,6 +294,31 @@ func (m *metaManager) processQuery(message model.Message) {
 	}
 }
 
+func (m *metaManager) processRemoteInsert(message model.Message) {
```

Comment:
we already use sendSync to send message, so can we just send the message to the cloud and when the cloud send resp
message, the sendSync will get the message. like podstatus